joomla/joomla-cms
# Media form field bootstrapped with static preview

Pull Request for Issue #10800 
#### Summary of Changes

Restore the static preview as an alternative to the pop over

Just a note here:
#### THIS AFFECTS CODE THAT IS NOT YET ACCEPTED IN CORE JOOMLA
#### Testing Instructions

Apply this PR
edit administrator/components/com_content/models/forms/article.xml and replace the intro image field with:

``` XML
        <field
            name="image_intro"
            type="media"
            label="COM_CONTENT_FIELD_INTRO_LABEL"
            description="COM_CONTENT_FIELD_INTRO_DESC"
            preview="true"
            preview_width="300"
            preview_height="300"
        />
```

edit an article and observe the changes!

PS for maintainers this also includes the #10788 which is already RTC
